src/Command/GenerateArticleCommand.php

```diff
@@ -43,7 +44,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $io->title('Création d\'un nouvel article');
 
-        $title = $io->ask('Titre');
+        $title = $io->ask('Titre', null, function ($value) {
```

Comment:
Quelque chose comme:

```php
use \Symfony\Component\Validator\Validation;

$title = $io->ask('Titre', null, Validation::createCallable(new NotBlank(['message' => 'Le titre est obligatoire.']));
```

est possible maintenant ✨

Reply:
Ref: https://symfony.com/blog/new-in-symfony-5-1-validator-improvements#validation-callables

Reply:
👌